acceptance/setup/pre_suite/15_setup_repos.rb

```diff
@@ -1,6 +1,14 @@
 def initialize_repo_on_host(host, os, nightly)
   case os
   when :debian
+
+    # For openjdk8
+    if host.platform.start_with?('debian-8')
```

Comment:
Rather than doing `platform.start-with` you can probably do `platform.variant` and `platform.version` - that seems like it might be a bit more robust? e.g. https://github.com/puppetlabs/puppetserver/pull/1383/files#diff-b1f26fc14676f1a802179a822fdd9377R202